File: src/main/java/com/weeth/domain/penalty/application/exception/AutoPenaltyDeleteNotAllowedException.java
package com.weeth.domain.penalty.application.exception;

import com.weeth.global.common.exception.BusinessLogicException;

public class AutoPenaltyDeleteNotAllowedException extends BusinessLogicException {
    public AutoPenaltyDeleteNotAllowedException() {
        super(PenaltyErrorCode.AUTO_PENALTY_DELETE_NOT_ALLOWED);
    }
}

File: src/main/kotlin/com/weeth/domain/penalty/application/exception/AutoPenaltyDeleteNotAllowedException.kt
package com.weeth.domain.penalty.application.exception;

import com.weeth.global.common.exception.BaseException;

public class AutoPenaltyDeleteNotAllowedException extends BaseException {
    public AutoPenaltyDeleteNotAllowedException() {
        super(PenaltyErrorCode.AUTO_PENALTY_DELETE_NOT_ALLOWED);
    }
}
